docs/developer-guide-commands.md

```diff
@@ -0,0 +1,109 @@
+# Development Helpers
+
+This document serves as a reference point to development helpers available under
+this project.
+
+## Building
+
+```shell
+# If pkg/apis/ has been changed, run generator to update client code
+make gen-k8s
+
+# Build handler operator (binary and docker)
+make handler
+```
+
+## Testing
+
+```shell
+# Run unit tests
+make test/unit
+
+# Run e2e tests
+make test/e2e
```

Comment:
please add a comment here that for this tests you need to have a k8s/openshift cluster running and also kubernetes-nmstate deploy on the cluster

Reply:
Done